spec/board-spec.rb

```diff
@@ -0,0 +1,70 @@
+require '../lib/game.rb'
+
+describe '::Board' do
+  let(:board) {Board.new}
+  let(:show_board) {board.show_board}
+  
+  
+  describe '#show_board' do
+
+    it 'prints the first row' do
+      show_board
```

Comment:
It's not suggested to print in testing. I suggest you remove `show_board`.

Reply:
Noted with thanks.